src/main/java/banking/database/Operations.java

```diff
@@ -0,0 +1,149 @@
+package banking.database;
+
+import banking.processing.Account;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class Operations {
+
+    private final String url;
+    private final String INSERT_INTO = "INSERT INTO card (number, pin) VALUES (?, ?)";
+    private final String SELECT_CARDNUMBER = "SELECT count(number) FROM card WHERE number = (?)";
+    private final String SELECT_ALL = "Select * FROM card";
+    private final String SELECT_ID = "SELECT id FROM card WHERE number=? AND pin=?";
+    private final String GET_CARD = "SELECT * FROM card WHERE id = ?";
+    private final String UPD_BALANCE = "UPDATE card SET balance = (balance + ?) WHERE id = ?";
+    private final String GET_MONEY = "UPDATE card SET balance = (balance + ?) WHERE number = ?";
+    private final String SEND_MONEY = "UPDATE card SET balance = (balance - ?) WHERE id = ?";
+    private final String DEL_ACC = "DELETE FROM card WHERE id = ?";
+
+    public Operations(String url) {
+        this.url = url;
+    }
+
+    public void insertCard(Account account) {
+        try (Connection conn = DriverManager.getConnection(url);
+             PreparedStatement preparedStatement = conn.prepareStatement(INSERT_INTO)) {
```

Comment:
Can you use the same approach to naming conn and preparedStatement?
You can use
```
Connection connection and PreparedStatement preparedStatement
or
Connection connection and PreparedStatement statement
```

Reply:
fixed